### DISCUSSION TITLE
Sentry

### DISCUSSION THREAD

**Discussion by fmotalleb** [selfhst/icons#141] [New Icon Request]
![sentry-glyph-dark-512x451](https://github.com/user-attachments/assets/e29e99f5-5857-471a-9b84-df22546ec715)
from: [sentry](https://sentry.io/branding/)

**Comment by shollyethan** [comment]
Sentry is already in the collection, but I went ahead and updated the non-light icon to match the purple icon you linked to in place of the black version I had previously been using. You may need to clear you browser cache to see the changes.
